src/Spec/Swagger2.php

```diff
@@ -227,6 +227,7 @@ public function getMethods($service)
                                 $output['parameters']['header'][] = $param;
                                 break;
                             case 'path':
+                                $param['enum'] = $parameter['enum'] ?? null;
```

Comment:
Is this done according to the spec? is our swagger2 file still valid?

Reply:
Yes it is 👍  There are a few warnings, we will address those